src/tree/FunctionAppTreeItem.ts

```diff
@@ -40,7 +40,7 @@ export class FunctionAppTreeItem implements IAzureParentTreeItem {
     }
 
     public get label(): string {
-        return this._state === 'Running' ? this.siteWrapper.name : `${this.siteWrapper.name} (${this._state})`;
+        return this.state === 'Running' ? this.siteWrapper.name : `${this.siteWrapper.name} (${this.state})`;
```

Comment:
I think Jay may only be showing when not running?  Can you make sure we're consistent across extensions?

Reply:
What if it's empty or undefined?

Reply:
This also only shows state when it's not running...I don't see how it's different than Jay's...

I don't know why it would ever be empty or undefined?